framework/data_request.go

```diff
@@ -62,9 +80,11 @@ func (f *framework) DataRequest(ctx context.Context, toID uint64, method string,
 		input:  input,
 		method: method,
 	}:
```

Comment:
Finally we locate the issue here. In both of the two running tasks, it blocks at pushing the dataRequest event to the channel. @fengjingchao  Any suggestion?